projects/gigya/README.md

```diff
@@ -0,0 +1,5 @@
+# Gigya
+
+Gigya library provides capabilities to integrate with SAP Customer Data Cloud from Gigya for enhanced identity management capabilities.
+
+`import { GigyaModule } from '@spartacus/gigya';`
```

Comment:
You can remove that line, as it will be described in details in the docs, how to enable it and configure.